x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/translations.ts

```diff
@@ -69,3 +69,10 @@ export const DELETED_RULE = i18n.translate(
     defaultMessage: 'Deleted rule',
   }
 );
+
+export const DELETE_CONFIRMATION_BODY = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleDetails.deleteRuleConfirmationBody',
+  {
+    defaultMessage: 'This action will delete the rule. Click "Confirm" to continue.',
```

Comment:
nit: Same as the other two i18n comments, might want to re-word w/ `Delete` as the action.